### PR TITLE
clarify that eth1 block follow distance for genesis

### PR DESCRIPTION
**Pull request by djrtwo**
addresses #1832

### CODE REVIEW

**Comment by djrtwo** [comment]
cc: @q9f @paulhauner @nisdas

File: specs/phase0/beacon-chain.md [hunk 1]
```diff
@@ -219,6 +220,7 @@ The following values are (non-configurable) constants used throughout the specif
 | - | - | :-: | :-: |
 | `MIN_GENESIS_DELAY` | `86400` | seconds | 1 day |
 | `SECONDS_PER_SLOT` | `12` | seconds | 12 seconds |
+| `SECONDS_PER_ETH1_BLOCK` | `14` | seconds | |
```
Comment (protolambda) [inline specs/phase0/beacon-chain.md:223]:
```suggestion
| `SECONDS_PER_ETH1_BLOCK` | `14` | seconds | 14 seconds |
```
It's not pretty, but at least consistent with the line above like this.

File: specs/phase0/beacon-chain.md [hunk 2]
```diff
@@ -183,6 +183,7 @@ The following values are (non-configurable) constants used throughout the specif
 
 | Name | Value |
 | - | - |
+| `ETH1_FOLLOW_DISTANCE` | `2**10` (= 1,024) |
```
Comment (protolambda) [inline specs/phase0/beacon-chain.md:186]: Now we are not expressing anymore that the distance is a distance of blocks. But I think it's clear enough from its use in the spec. Maybe with the descriptions PR we can add more comments for each constant.

**Comment by protolambda** [comment]
This PR is duplicate (kind of)! It does introduce some refinement, but I also didn't like the wording where it would seemingly only be a single consecutive chain if `step ==  1`, and not necessarily otherwise (from one server at least, with gaps).

The PR: https://github.com/ethereum/eth2.0-specs/pull/1795/files

File: specs/phase0/beacon-chain.md (hunk 1 above)
Comment (djrtwo) [reply specs/phase0/beacon-chain.md:223]: lol that is a funny column

File: specs/phase0/beacon-chain.md (hunk 2 above)
Comment (djrtwo) [reply specs/phase0/beacon-chain.md:186]: Agreed. I wasn't too happy with how it looks here.
I also considered just making the eth2 follow distance purely in seconds and getting rid of this constant but (1) you miss the meaning of ~1024 block  and (2) I'd rather leave the constants stable for implementers

**Comment by q9f** [comment]
Thanks

**Pull request merged**